File: C Programs/Week1/merge.c
#include <stdio.h>
int main() {
    int a[50], b[50], merged[100];
    int n1, n2;
    printf("Enter size of first array: ");
    scanf("%d", &n1);
    printf("Enter elements of first array:\n");
    for (int i = 0; i < n1; i++) scanf("%d", &a[i]);
    printf("Enter size of second array: ");
    scanf("%d", &n2);
    printf("Enter elements of second array:\n");
    for (int i = 0; i < n2; i++) scanf("%d", &b[i]);
    for (int i = 0; i < n1; i++) merged[i] = a[i];
    for (int i = 0; i < n2; i++) merged[n1 + i] = b[i];
    printf("Merged array: ");
    for (int i = 0; i < n1 + n2; i++) printf("%d ", merged[i]);
    return 0;
}

File: Python Programs/Week1/merge.py
n=int(input("Enter first array length: "))
m=int(input("Enter second array length: "))
a1=[]
a2=[]
print("Enter first array elements: ")
for i in range(n):
    e=int(input())
    a1.append(e)
print("First array: ")
for i in range(n):
    print(str(a1[i])+" ", end=" ")
print()
print("Enter second array elements: ")
for i in range(m):
    e=int(input())
    a2.append(e)
print("Second array: ")
for i in range(m):
    print(str(a2[i])+" ", end=" ")
print()
print("Merged arrays: ")
a1=a1+a2
print()
for i in range(n+m):
    print(str(a1[i])+" ",end=" ")
print()